modules/cnv-configuring-vmi-live-migration-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-configuring-vmi-eviction-strategy.adoc
+
+[id="cnv-configuring-vmi-live-migration-cli_{context}"]
+= Configuring custom virtual machine instances with the `LiveMigration` 
+eviction strategy 
+
+The `LiveMigrate` eviction strategy ensures that a virtual machine instance is 
+live migrated to another node in the case of a node being placed in 
+maintenance mode, or if the `node drain` command is performed on the node. 
+
+Only custom virtual machines need to have this feature manually configured. This
+ eviction strategy is configured by default in virtual machines created from 
+common templates. 
+
+.Prerequisites
+
+* The live migration feature must be enabled in the cluster.
```

Comment:
Right you are. This is for #14923 (node maintenance). I mistakenly added this instead of the live migration concept that is shared across both that PR and this one. I've fixed that now, and will add the feedback you've noted for this topic in the 14923.